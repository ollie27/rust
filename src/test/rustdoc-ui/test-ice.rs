// compile-flags:--test -Z unstable-options -Z treat-err-as-bug
// failure-status: 101

fail
