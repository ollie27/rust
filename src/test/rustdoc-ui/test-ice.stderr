error: expected one of `!` or `::`, found `<eof>`
 --> $DIR/test-ice.rs:3:1
  |
3 | fail
  | ^^^^ expected one of `!` or `::` here

error: aborting due to previous error

